_search-plugins/knn/approximate-knn.md

```diff
@@ -363,6 +367,11 @@ Not every method supports each of these spaces. Be sure to check out [the method
         \[ \text{If} d > 0, score = d + 1 \] \[\text{If} d \le 0\] \[score = {1 \over 1 + (-1 &middot; d) }\]
     </td>
   </tr>
+  <tr>
+    <td>hammingbit</td>
```

Comment:
Can we mention that this works only with binary data type and other space type does not work with binary data type?

Reply:
ack

Reply:
Added note at bottom